src/ucp/core/ucp_types.h

```diff
@@ -67,6 +67,9 @@ enum {
     UCP_AM_ID_RNDV_DATA_LAST    =  13, /* The last rndv data fragment when using
                                           software rndv (bcopy) */
     UCP_AM_ID_OFFLOAD_SYNC_ACK  =  14, /* Eager sync ack for tag offload proto */
+
+    UCP_AM_ID_EAGER_STREAM      =  15, /* Eager STREAM packet */
```

Comment:
we better not use "eager" terminology with stream. What about just "UCP_AM_ID_STREAM_DATA"?